### PR TITLE
feat(Icons)!: Streamline Icon Set Full [GM-30]

### PR DESCRIPTION
**Pull request by codecaaron**
## Overview
Streamline your life.

- [Storybook](https://5ef25401a3ef6801e030954b--codecademy-styleguide.netlify.app)
- [Monolith](https://github.com/codecademy-engineering/Codecademy/pull/18095)
- [Static Sites](https://github.com/codecademy-engineering/static-sites/pull/176)
- [Zoo](https://caracal.codecademy.com)

### Changes
- Fully overwrites all gamut-icons svgs with new streamline assets.
- Removes title-id and aria-labeled by default (these are implementable on the component but it will no longer cause an accessibility ID collision when 2 of the same icon are on the page at once.  (https://www.scottohara.me/blog/2019/05/22/contextual-images-svgs-and-a11y.html#svgs-that-convey-information)
- Removes some remaining usages of svg/icons in components local to gamut with a few dynamic exceptions.
- Moves Ikona and general Icons to a specific folder that is not exported on index.
- Also adds an eslint rule that will throw and error if you import from anything in `/dist/deprecated`

### PR Checklist

- [x] Related to Abstract designs: Everywhere
- [x] Related to JIRA ticket: WEB-30
- [x] I have run this code to verify it works
- [ ] This PR includes unit tests for the code change

---

## Merging your changes

When you are ready to merge to master and publish your changes, use the "squash and merge" button in GitHub, and follow the [PR Title Guide](https://github.com/Codecademy/client-modules#pr-title-guide) to format your PR title and write your PR merge description.

**IMPORTANT:** If your PR contains breaking changes, please remember to follow the instructions for breaking changes!

### CODE REVIEW

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-icons@2.4.1-alpha.70803a.0<br/>@codecademy/gamut-labs@1.7.24-alpha.70803a.0<br/>@codecademy/gamut@9.0.6-alpha.70803a.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5ee9064700e2e80206c7e4d7--codecademy-styleguide.netlify.app>https://5ee9064700e2e80206c7e4d7--codecademy-styleguide.netlify.app</a></p><p><a href=https://app.netlify.com/sites/codecademy-styleguide/deploys/5ee9064700e2e80206c7e4d7>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-icons@2.4.1-alpha.553015.0<br/>@codecademy/gamut-labs@1.7.24-alpha.553015.0<br/>@codecademy/gamut@9.0.6-alpha.553015.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5eeb7af7167a9becce38fd6c--codecademy-styleguide.netlify.app>https://5eeb7af7167a9becce38fd6c--codecademy-styleguide.netlify.app</a></p><p><a href=https://app.netlify.com/sites/codecademy-styleguide/deploys/5eeb7af7167a9becce38fd6c>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-icons@2.4.1-alpha.a8022e.0<br/>@codecademy/gamut-labs@1.7.24-alpha.a8022e.0<br/>@codecademy/gamut@9.0.6-alpha.a8022e.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5ef09dfb05ba267fdeeb3b34--codecademy-styleguide.netlify.app>https://5ef09dfb05ba267fdeeb3b34--codecademy-styleguide.netlify.app</a></p><p><a href=https://app.netlify.com/sites/codecademy-styleguide/deploys/5ef09dfb05ba267fdeeb3b34>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-icons@2.4.1-alpha.0dcafb.0<br/>@codecademy/gamut-labs@1.7.24-alpha.0dcafb.0<br/>@codecademy/gamut@9.0.6-alpha.0dcafb.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5ef0a13d1b1f3218f274d54d--codecademy-styleguide.netlify.app>https://5ef0a13d1b1f3218f274d54d--codecademy-styleguide.netlify.app</a></p><p><a href=https://app.netlify.com/sites/codecademy-styleguide/deploys/5ef0a13d1b1f3218f274d54d>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-icons@2.4.1-alpha.bdc517.0<br/>@codecademy/gamut-labs@1.7.24-alpha.bdc517.0<br/>@codecademy/gamut@9.0.6-alpha.bdc517.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5ef0c0b305ba26e7afeb3044--codecademy-styleguide.netlify.app>https://5ef0c0b305ba26e7afeb3044--codecademy-styleguide.netlify.app</a></p><p><a href=https://app.netlify.com/sites/codecademy-styleguide/deploys/5ef0c0b305ba26e7afeb3044>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-icons@2.4.1-alpha.ffe1c5.0<br/>@codecademy/gamut-labs@1.7.24-alpha.ffe1c5.0<br/>@codecademy/gamut@9.0.6-alpha.ffe1c5.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5ef0e0befaa85a5eae53ffb5--codecademy-styleguide.netlify.app>https://5ef0e0befaa85a5eae53ffb5--codecademy-styleguide.netlify.app</a></p><p><a href=https://app.netlify.com/sites/codecademy-styleguide/deploys/5ef0e0befaa85a5eae53ffb5>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-icons@2.4.1-alpha.06a44c.0<br/>@codecademy/gamut-labs@1.7.24-alpha.06a44c.0<br/>@codecademy/gamut@9.0.6-alpha.06a44c.0<br/></pre>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-icons@2.4.1-alpha.fb67b3.0<br/>@codecademy/gamut-labs@1.7.24-alpha.fb67b3.0<br/>@codecademy/gamut@9.0.6-alpha.fb67b3.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5ef0e7e84800ac01c495527d--codecademy-styleguide.netlify.app>https://5ef0e7e84800ac01c495527d--codecademy-styleguide.netlify.app</a></p><p><a href=https://app.netlify.com/sites/codecademy-styleguide/deploys/5ef0e7e84800ac01c495527d>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-icons@2.4.1-alpha.03929e.0<br/>@codecademy/gamut-labs@1.7.29-alpha.03929e.0<br/>@codecademy/gamut@9.2.2-alpha.03929e.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5ef11205e5a1b402a00510a5--codecademy-styleguide.netlify.app>https://5ef11205e5a1b402a00510a5--codecademy-styleguide.netlify.app</a></p><p><a href=https://app.netlify.com/sites/codecademy-styleguide/deploys/5ef11205e5a1b402a00510a5>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-icons@2.4.1-alpha.1928b4.0<br/>@codecademy/gamut-labs@1.7.29-alpha.1928b4.0<br/>@codecademy/gamut@9.2.2-alpha.1928b4.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5ef1552d39364e11bee4443a--codecademy-styleguide.netlify.app>https://5ef1552d39364e11bee4443a--codecademy-styleguide.netlify.app</a></p><p><a href=https://app.netlify.com/sites/codecademy-styleguide/deploys/5ef1552d39364e11bee4443a>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-icons@2.4.1-alpha.2ca365.0<br/>@codecademy/gamut-labs@1.7.29-alpha.2ca365.0<br/>@codecademy/gamut@9.2.2-alpha.2ca365.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5ef25401a3ef6801e030954b--codecademy-styleguide.netlify.app>https://5ef25401a3ef6801e030954b--codecademy-styleguide.netlify.app</a></p><p><a href=https://app.netlify.com/sites/codecademy-styleguide/deploys/5ef25401a3ef6801e030954b>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-icons@2.4.2-alpha.362a47.0<br/>@codecademy/gamut-labs@1.7.30-alpha.362a47.0<br/>@codecademy/gamut@9.2.3-alpha.362a47.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5ef34e1ba80f876b26d258c6--codecademy-styleguide.netlify.app>https://5ef34e1ba80f876b26d258c6--codecademy-styleguide.netlify.app</a></p><p><a href=https://app.netlify.com/sites/codecademy-styleguide/deploys/5ef34e1ba80f876b26d258c6>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/eslint-config@0.3.4-alpha.3340e8.0<br/>@codecademy/gamut-icons@2.4.2-alpha.3340e8.0<br/>@codecademy/gamut-labs@1.7.30-alpha.3340e8.0<br/>@codecademy/gamut@9.2.3-alpha.3340e8.0<br/></pre>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/eslint-config@0.3.4-alpha.bb9b40.0<br/>@codecademy/gamut-icons@2.4.2-alpha.bb9b40.0<br/>@codecademy/gamut-labs@1.7.30-alpha.bb9b40.0<br/>@codecademy/gamut@9.2.3-alpha.bb9b40.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5ef37715405975024590eb4d--codecademy-styleguide.netlify.app>https://5ef37715405975024590eb4d--codecademy-styleguide.netlify.app</a></p><p><a href=https://app.netlify.com/sites/codecademy-styleguide/deploys/5ef37715405975024590eb4d>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/eslint-config@0.3.4-alpha.aff0ad.0<br/>@codecademy/gamut-icons@2.4.2-alpha.aff0ad.0<br/>@codecademy/gamut-labs@1.7.30-alpha.aff0ad.0<br/>@codecademy/gamut@9.2.3-alpha.aff0ad.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5ef37cd6ef66f6cf7b881429--codecademy-styleguide.netlify.app>https://5ef37cd6ef66f6cf7b881429--codecademy-styleguide.netlify.app</a></p><p><a href=https://app.netlify.com/sites/codecademy-styleguide/deploys/5ef37cd6ef66f6cf7b881429>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/eslint-config@0.3.4-alpha.b1d68a.0<br/>@codecademy/gamut-icons@2.4.2-alpha.b1d68a.0<br/>@codecademy/gamut-labs@1.7.30-alpha.b1d68a.0<br/>@codecademy/gamut@9.2.3-alpha.b1d68a.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5ef3810d994fee0149b9b02d--codecademy-styleguide.netlify.app>https://5ef3810d994fee0149b9b02d--codecademy-styleguide.netlify.app</a></p><p><a href=https://app.netlify.com/sites/codecademy-styleguide/deploys/5ef3810d994fee0149b9b02d>Deploy Logs</a></p>

File: packages/gamut-icons/src/svg/arrow-chevron-down-icon.svg [hunk 1]
```diff
@@ -1 +1 @@
-<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><defs><style>.cls-1{fill:none;stroke:currentColor;stroke-linecap:round;stroke-linejoin:round;stroke-width:1.5px;fill-rule:evenodd;}</style></defs><title>arrow-down-1</title><path class="cls-1" d="M23.25,7.311,12.53,18.03a.749.749,0,0,1-1.06,0L.75,7.311"/></svg>
+<svg id="Regular" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><defs><style>.cls-1{fill:none;stroke:currentColor;stroke-linecap:round;stroke-linejoin:round;stroke-width:1.5px;fill-rule:evenodd;}</style></defs><title>arrow-down-1</title><path class="cls-1" d="M23.25,7.311,12.53,18.03a.749.749,0,0,1-1.06,0L.75,7.311"/></svg>
```
Comment (JoshuaKGoldberg) [inline packages/gamut-icons/src/svg/arrow-chevron-down-icon.svg:1]: > `id="Regular"`

Hmm... _hmm_... remove?

File: packages/gamut-icons/src/svg/file-add-icon.svg [hunk 2]
```diff
@@ -1,6 +1,20 @@
-<svg viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg">
-    <g fill="none" fill-rule="evenodd">
-        <path d="M4 0h11.172a2 2 0 011.414.586L19 3l2.414 2.414A2 2 0 0122 6.828V22a2 2 0 01-2 2H4a2 2 0 01-2-2V2a2 2 0 012-2zm1 2a1 1 0 00-1 1v18a1 1 0 001 1h14a1 1 0 001-1V8h-5a1 1 0 01-1-1V2H5zm11 4h3l-1.5-1.5L16 3v3z" fill="#111"/>
-        <path d="M9 15h6m-3-3v6" stroke="#111" stroke-linecap="round" stroke-width="2"/>
-    </g>
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Generator: Adobe Illustrator 22.1.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
```
Comment (JoshuaKGoldberg) [inline packages/gamut-icons/src/svg/file-add-icon.svg:2]:
```suggestion
```

Up for getting rid of all these? 😄

File: packages/gamut/src/index.tsx [hunk 3]
```diff
@@ -44,76 +42,3 @@ export * from './Truncate';
 export * from './utils';
 export * from './Video';
 export * from './typings/responsive-properties';
-export * from './Icon/icons/AccessibilityIcon';
```
Comment (JoshuaKGoldberg) [inline packages/gamut/src/index.tsx:47]: These being moved to deprecated make this a breaking change PR, right?

**Comment by JoshuaKGoldberg** [comment]
I like the strategy in this PR a lot @codecaaron. Really nicely done! Especially removing the automatic icon label generation, now that we have aXe tests running on the majority of monolith site pages -> and will be forced to do these more intentionally. Perpetual +1 from me on ESLint rules 😄 .

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/eslint-config@0.3.4-alpha.179394.0<br/>@codecademy/gamut-icons@2.4.2-alpha.179394.0<br/>@codecademy/gamut-labs@1.7.31-alpha.179394.0<br/>@codecademy/gamut@9.2.4-alpha.179394.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5ef3d51063caac8e153612d6--codecademy-styleguide.netlify.app>https://5ef3d51063caac8e153612d6--codecademy-styleguide.netlify.app</a></p><p><a href=https://app.netlify.com/sites/codecademy-styleguide/deploys/5ef3d51063caac8e153612d6>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/eslint-config@0.3.4-alpha.0c743e.0<br/>@codecademy/gamut-icons@2.4.2-alpha.0c743e.0<br/>@codecademy/gamut-labs@1.7.31-alpha.0c743e.0<br/>@codecademy/gamut@9.2.4-alpha.0c743e.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5ef3dfb262a7f78674a7482a--codecademy-styleguide.netlify.app>https://5ef3dfb262a7f78674a7482a--codecademy-styleguide.netlify.app</a></p><p><a href=https://app.netlify.com/sites/codecademy-styleguide/deploys/5ef3dfb262a7f78674a7482a>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/eslint-config@0.3.4-alpha.62664d.0<br/>@codecademy/gamut-icons@2.4.2-alpha.62664d.0<br/>@codecademy/gamut-labs@1.7.31-alpha.62664d.0<br/>@codecademy/gamut@9.2.4-alpha.62664d.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5ef4bb035ac5bc889b7b5253--codecademy-styleguide.netlify.app>https://5ef4bb035ac5bc889b7b5253--codecademy-styleguide.netlify.app</a></p><p><a href=https://app.netlify.com/sites/codecademy-styleguide/deploys/5ef4bb035ac5bc889b7b5253>Deploy Logs</a></p>

File: packages/styleguide/stories/Core/Atoms/Icons/Icons.stories.mdx [hunk 4]
```diff
@@ -48,6 +48,18 @@ import { selectableColors } from '../../../../helpers';
...
 <Props of={AlertIcon} />
 
+## Usage
+
+```tsx
+// Index
+import { AlertIcon } from '@codecademy/gamut-icons';
+
+// Direct Path
+import AlertIconfrom '@codecademy/gamut-icons/dist/AlertIcon';
```
Comment (jakemhiller) [inline packages/styleguide/stories/Core/Atoms/Icons/Icons.stories.mdx:58]: Do we want to tell ppl to do this?

also, you are missing a space in `import AlertIconfrom`

Comment (codecaaron) [reply packages/styleguide/stories/Core/Atoms/Icons/Icons.stories.mdx]: Good point.

**Comment by codecaaron** [comment]
> Can't comment on the file directly but can we keep/update the [rocket icon](https://github.com/Codecademy/client-modules/pull/838/files#diff-b909c4954a62aa35f29d43b24add399b) instead of deleting? I believe it comes from this new set, Sarah sent me the asset in the last couple weeks. We're using 2 actual Gamut icons in Author (also UserManagementIcon, but I can swap that one out for the new one here). I'd love to be able to swap out Author's icons for these new ones.

Added back the Rocket.  If you like I can make the PR for the Icon switch.  A full switch will be a bit wonky since its almost all from `LocalGamut`

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/eslint-config@0.3.4-alpha.6b7bda.0<br/>@codecademy/gamut-icons@2.4.2-alpha.6b7bda.0<br/>@codecademy/gamut-labs@1.7.31-alpha.6b7bda.0<br/>@codecademy/gamut@9.2.4-alpha.6b7bda.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5ef4cf418a5bc8e1f5beb052--codecademy-styleguide.netlify.app>https://5ef4cf418a5bc8e1f5beb052--codecademy-styleguide.netlify.app</a></p><p><a href=https://app.netlify.com/sites/codecademy-styleguide/deploys/5ef4cf418a5bc8e1f5beb052>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/eslint-config@0.3.4-alpha.4f65a5.0<br/>@codecademy/gamut-icons@2.4.2-alpha.4f65a5.0<br/>@codecademy/gamut-labs@1.7.31-alpha.4f65a5.0<br/>@codecademy/gamut@9.2.4-alpha.4f65a5.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5f04b10eb2922a02594a43a3--codecademy-styleguide.netlify.app>https://5f04b10eb2922a02594a43a3--codecademy-styleguide.netlify.app</a></p><p><a href=https://app.netlify.com/sites/codecademy-styleguide/deploys/5f04b10eb2922a02594a43a3>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/eslint-config@0.3.4-alpha.6cde78.0<br/>@codecademy/gamut-icons@2.4.2-alpha.6cde78.0<br/>@codecademy/gamut-labs@1.7.36-alpha.6cde78.0<br/>@codecademy/gamut@9.2.9-alpha.6cde78.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5f060d60062816375ec03261--codecademy-styleguide.netlify.app>https://5f060d60062816375ec03261--codecademy-styleguide.netlify.app</a></p><p><a href=https://app.netlify.com/sites/codecademy-styleguide/deploys/5f060d60062816375ec03261>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/eslint-config@0.3.4-alpha.d4779c.0<br/>@codecademy/gamut-icons@2.4.2-alpha.d4779c.0<br/>@codecademy/gamut-labs@1.7.36-alpha.d4779c.0<br/>@codecademy/gamut@9.2.9-alpha.d4779c.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5f0619412d3ff6e8be1f1491--codecademy-styleguide.netlify.app>https://5f0619412d3ff6e8be1f1491--codecademy-styleguide.netlify.app</a></p><p><a href=https://app.netlify.com/sites/codecademy-styleguide/deploys/5f0619412d3ff6e8be1f1491>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/eslint-config@0.3.4-alpha.cb9cd9.0<br/>@codecademy/gamut-icons@2.4.2-alpha.cb9cd9.0<br/>@codecademy/gamut-labs@1.7.42-alpha.cb9cd9.0<br/>@codecademy/gamut@9.5.2-alpha.cb9cd9.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5f10b45a3977d26b711229d4--codecademy-styleguide.netlify.app>https://5f10b45a3977d26b711229d4--codecademy-styleguide.netlify.app</a></p><p><a href=https://app.netlify.com/sites/codecademy-styleguide/deploys/5f10b45a3977d26b711229d4>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/eslint-config@0.3.4-alpha.55894f.0<br/>@codecademy/gamut-icons@2.4.2-alpha.55894f.0<br/>@codecademy/gamut-labs@1.7.42-alpha.55894f.0<br/>@codecademy/gamut@9.5.2-alpha.55894f.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5f10cceb108a39a5b3bff8ff--codecademy-styleguide.netlify.app>https://5f10cceb108a39a5b3bff8ff--codecademy-styleguide.netlify.app</a></p><p><a href=https://app.netlify.com/sites/codecademy-styleguide/deploys/5f10cceb108a39a5b3bff8ff>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/eslint-config@0.3.4-alpha.757bf7.0<br/>@codecademy/gamut-icons@2.4.2-alpha.757bf7.0<br/>@codecademy/gamut-labs@1.7.42-alpha.757bf7.0<br/>@codecademy/gamut@9.5.2-alpha.757bf7.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5f10cef433fc6594afa7f49d--codecademy-styleguide.netlify.app>https://5f10cef433fc6594afa7f49d--codecademy-styleguide.netlify.app</a></p><p><a href=https://app.netlify.com/sites/codecademy-styleguide/deploys/5f10cef433fc6594afa7f49d>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5f10d0f53de256025cfed158--codecademy-styleguide.netlify.app>https://5f10d0f53de256025cfed158--codecademy-styleguide.netlify.app</a></p><p><a href=https://app.netlify.com/sites/codecademy-styleguide/deploys/5f10d0f53de256025cfed158>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/eslint-config@0.3.4-alpha.c794d8.0<br/>@codecademy/gamut-icons@2.4.2-alpha.c794d8.0<br/>@codecademy/gamut-labs@1.7.42-alpha.c794d8.0<br/>@codecademy/gamut@9.5.2-alpha.c794d8.0<br/></pre>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/eslint-config@0.3.4-alpha.3f4582.0<br/>@codecademy/gamut-icons@2.4.2-alpha.3f4582.0<br/>@codecademy/gamut-labs@1.7.42-alpha.3f4582.0<br/>@codecademy/gamut@9.5.2-alpha.3f4582.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5f10d5bf31789c022966397b--codecademy-styleguide.netlify.app>https://5f10d5bf31789c022966397b--codecademy-styleguide.netlify.app</a></p><p><a href=https://app.netlify.com/sites/codecademy-styleguide/deploys/5f10d5bf31789c022966397b>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/eslint-config@0.3.4-alpha.4d5375.0<br/>@codecademy/gamut-icons@2.4.2-alpha.4d5375.0<br/>@codecademy/gamut-labs@1.7.42-alpha.4d5375.0<br/>@codecademy/gamut@9.5.2-alpha.4d5375.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5f10d830e5bff102096067ad--codecademy-styleguide.netlify.app>https://5f10d830e5bff102096067ad--codecademy-styleguide.netlify.app</a></p><p><a href=https://app.netlify.com/sites/codecademy-styleguide/deploys/5f10d830e5bff102096067ad>Deploy Logs</a></p>

File: packages/eslint-config/index.js [hunk 5]
```diff
@@ -106,6 +106,14 @@ module.exports = {
 
     // These style rules conflict with Prettier but aren't disabled by its plugins
     '@typescript-eslint/quotes': 'off',
+
+    'no-restricted-imports': [
+      'error',
+      {
+        paths: ['@codecademy/gamut/dist/deprecated'],
```
Comment (JoshuaKGoldberg) [inline packages/eslint-config/index.js:113]: Ooh maybe `@codecademy/*/dist/*` (or whatever version of that works)? We'll want to block `gamut-labs` too...

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/eslint-config@0.3.4-alpha.c6ad6f.0<br/>@codecademy/gamut-icons@2.4.2-alpha.c6ad6f.0<br/>@codecademy/gamut-labs@1.7.42-alpha.c6ad6f.0<br/>@codecademy/gamut@9.5.2-alpha.c6ad6f.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5f11b9264efb9ecebf44c061--codecademy-styleguide.netlify.app>https://5f11b9264efb9ecebf44c061--codecademy-styleguide.netlify.app</a></p><p><a href=https://app.netlify.com/sites/codecademy-styleguide/deploys/5f11b9264efb9ecebf44c061>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/eslint-config@0.3.4-alpha.a4d7b7.0<br/>@codecademy/gamut-icons@2.4.2-alpha.a4d7b7.0<br/>@codecademy/gamut-labs@1.7.44-alpha.a4d7b7.0<br/>@codecademy/gamut@9.5.4-alpha.a4d7b7.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5f11d10c6765e1019ace4a86--codecademy-styleguide.netlify.app>https://5f11d10c6765e1019ace4a86--codecademy-styleguide.netlify.app</a></p><p><a href=https://app.netlify.com/sites/codecademy-styleguide/deploys/5f11d10c6765e1019ace4a86>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/eslint-config@0.3.4-alpha.a67b7a.0<br/>@codecademy/gamut-icons@2.4.2-alpha.a67b7a.0<br/>@codecademy/gamut-labs@1.7.44-alpha.a67b7a.0<br/>@codecademy/gamut@9.5.4-alpha.a67b7a.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5f11d28634ceb507ed600e93--codecademy-styleguide.netlify.app>https://5f11d28634ceb507ed600e93--codecademy-styleguide.netlify.app</a></p><p><a href=https://app.netlify.com/sites/codecademy-styleguide/deploys/5f11d28634ceb507ed600e93>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/eslint-config@0.3.4-alpha.ee8bb8.0<br/>@codecademy/gamut-icons@2.4.2-alpha.ee8bb8.0<br/>@codecademy/gamut-labs@1.7.44-alpha.ee8bb8.0<br/>@codecademy/gamut@9.5.4-alpha.ee8bb8.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5f171b941a564e61b530de67--codecademy-styleguide.netlify.app>https://5f171b941a564e61b530de67--codecademy-styleguide.netlify.app</a></p><p><a href=https://app.netlify.com/sites/codecademy-styleguide/deploys/5f171b941a564e61b530de67>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/eslint-config@1.0.1-alpha.98018e.0<br/>@codecademy/gamut-icons@2.4.3-alpha.98018e.0<br/>@codecademy/gamut-labs@1.7.49-alpha.98018e.0<br/>@codecademy/gamut@11.1.1-alpha.98018e.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5f1861ed2690d301f50612aa--codecademy-styleguide.netlify.app>https://5f1861ed2690d301f50612aa--codecademy-styleguide.netlify.app</a></p><p><a href=https://app.netlify.com/sites/codecademy-styleguide/deploys/5f1861ed2690d301f50612aa>Deploy Logs</a></p>

**Pull request merged**